test/src/test/java/org/corfudb/integration/MultiStreamMergeStreamListenerIT.java

```diff
@@ -0,0 +1,300 @@
+        );
+
+        replicationStatusTable = store.openTable(LogReplicationMetadataManager.NAMESPACE,
+                LogReplicationMetadataManager.REPLICATION_STATUS_TABLE,
+                LogReplicationMetadata.ReplicationStatusKey.class,
+                LogReplicationMetadata.ReplicationStatusVal.class,
+                null,
+                TableOptions.fromProtoSchema(LogReplicationMetadata.ReplicationStatusVal.class));
+    }
+
+    private void writeUpdatesToDefaultTable(int numUpdates, int offset) throws Exception {
+
+        // Make some updates to tableA
+        for (int index = offset; index < offset + numUpdates; index++) {
+            // Update TableA
+            try (TxnContext tx = store.txn(namespace)) {
+                SampleSchema.Uuid uuid = SampleSchema.Uuid.newBuilder().setMsb(index).setLsb(index).build();
+                SampleSchema.SampleTableAMsg msgA = SampleSchema.SampleTableAMsg.newBuilder().setPayload(String.valueOf(index)).build();
+                tx.putRecord(tableA, uuid, msgA, uuid);
+                tx.commit();
+            }
+        }
+    }
+
+    private void writeUpdatesToBothTable(int numUpdates, int offset) throws Exception {
+
+        // Make some updates to tableA
+        for (int index = offset; index < offset + numUpdates; index++) {
+            if (index % 2 == 0) {
+                // Update TableA
+                try (TxnContext tx = store.txn(namespace)) {
+                    SampleSchema.Uuid uuid = SampleSchema.Uuid.newBuilder().setMsb(index).setLsb(index).build();
+                    SampleSchema.SampleTableAMsg msgA = SampleSchema.SampleTableAMsg.newBuilder().setPayload("tableA " + String.valueOf(index)).build();
+                    tx.putRecord(tableA, uuid, msgA, uuid);
+                    tx.commit();
+                }
+            } else {
+                // Update LRStatus table
+                LogReplicationMetadata.ReplicationStatusKey key = LogReplicationMetadata.ReplicationStatusKey
+                        .newBuilder().setClusterId("lrstatus" + Integer.toString(index)).build();
+                LogReplicationMetadata.ReplicationStatusVal val = LogReplicationMetadata.ReplicationStatusVal
+                        .newBuilder().setRemainingEntriesToSend(index).build();
+                try (TxnContext tx = store.txn(CORFU_SYSTEM_NAMESPACE)) {
+                    tx.putRecord(replicationStatusTable, key, val, null);
+                    tx.commit();
+                }
+            }
+        }
+    }
+
+    private void verifyUpdates(LinkedList<CorfuStreamEntries> updates) {
+        long startTs = -1;
+        for (CorfuStreamEntries update : updates) {
+            if(startTs == -1) {
+                startTs = update.getTimestamp().getSequence();
+                continue;
+            }
+            assertThat(startTs < update.getTimestamp().getSequence());
+            startTs = update.getTimestamp().getSequence();
+        }
+    }
+
+    /*
```

Comment:
nit: Indentation is a bit off

can we also reword the comment and simplify it?

Reply:
resolving outdated/obsolete comments